autoconnect/autoconnect-ws/autoconnect-ws-sm/src/identified/mod.rs

```diff
@@ -297,6 +297,8 @@ pub async fn process_existing_user(
                 .metrics
                 .incr_with_tags("ua.expiration")
                 .with_tag("errno", "104")
+                .with_tag("autoconnect", "true")
+                .with_tag("reason", "no_current_mo")
```

Comment:
None of the `autoconnect: true` are necessary, that's already emitted for all metrics. Might as well name these `_month`.

Reply:
Yeah, truncated these partly because that's about where they truncate on my display, and I felt it provided enough detail.